src/tsconfig.json

```diff
@@ -4,6 +4,7 @@
         "types": [],
         "baseUrl": "./",
         "rootDir": "./",
-        "outDir": "../.temp/manualBuildCore"
+        "outDir": "../.temp/manualBuildCore",
+        "downlevelIteration": true

```

Comment:
This might slow down iteration in general in babylon which I am afraid of.

Reply:
Removed.